src/system/SystemConfig.h

```diff
@@ -682,6 +682,20 @@ struct LwIPEvent;
 #endif
 #endif // CHIP_SYSTEM_CONFIG_USE_ZEPHYR_NET_IF
 
+#ifndef CHIP_SYSTEM_CONFIG_USE_MBED_NET_IF
```

Comment:
Is it used?

Reply:
Not anymore, initially we didn't had the bsd netif emulation in place and used special case for mbed.

Reply:
I removed it.